contrib/epee/src/readline_buffer.cpp

```diff
@@ -16,6 +17,8 @@ static std::string last_prompt;
 std::mutex line_mutex, sync_mutex, process_mutex;
 std::condition_variable have_line;
 
+std::vector<std::string> rdln::readline_buffer::completion_commands = {"exit"};
```

Comment:
I have no immediate plans to refactor. Readline has a horrible interface and the statics are needed unfortunately. Further complicated by the fact that all these patches have been designed to slot into the project with hardly any changes required to the rest of the code base (which trust me was tough!).